src/pod-security-policy/forbidden-sysctls/src.rego

```diff
@@ -1,11 +1,19 @@
 package k8spspforbiddensysctls
 
+# Block if forbidden
 violation[{"msg": msg, "details": {}}] {
     sysctl := input.review.object.spec.securityContext.sysctls[_].name
     forbidden_sysctl(sysctl)
     msg := sprintf("The sysctl %v is not allowed, pod: %v. Forbidden sysctls: %v", [sysctl, input.review.object.metadata.name, input.parameters.forbiddenSysctls])
 }
 
+# Block if not explicitly allowed
+violation[{"msg": msg, "details": {}}] {
+    sysctl := input.review.object.spec.securityContext.sysctls[_].name
+    not allowed_sysctl(sysctl)
+    msg := sprintf("The sysctl %v is not explictly allowed, pod: %v. Allowed sysctls: %v", [sysctl, input.review.object.metadata.name, input.parameters.allowedSysctls])
+}
+
```

Comment:
It looks like rego is being updated here. Please bump the version as per: https://docs.google.com/document/d/1IYiypA-mRcdfSVfmoeyuaeG8XtA1u4GkcqH3kEkv2uw/edit#

Reply:
cc @maxsmythe @ritazh @sozercan

Reply:
+1, because we're adding a new parameter field, we should bump the minor version

Reply:
Updated the minor version in 5108082aaa6b8b03bad20aff77889eb997fce7d2